arrow-string/src/length.rs

```diff
@@ -137,6 +138,22 @@ pub fn bit_length(array: &dyn Array) -> Result<ArrayRef, ArrowError> {
             let list = array.as_string::<i64>();
             Ok(bit_length_impl::<Int64Type>(list.offsets(), list.nulls()))
         }
+        DataType::Utf8View => {
+            let list = array.as_string_view();
+            let values = (0..list.len())
+                .map(|i| {
+                    if list.is_valid(i) {
+                        list.views()[i] as u32 * 8
+                    } else {
+                        0
+                    }
+                })
+                .collect::<Vec<u32>>();
+            Ok(Arc::new(UInt32Array::new(
+                values.into(),
```

Comment:
```suggestion
            let values = list.views().iter().map(|view| (view as i32).wrapping_mul(8)).collect();
            Ok(Arc::new(UInt32Array::new(
                values,
```

As we're doing infallible wrapping arithmetic, we can just compute this for everything, this will vectorize better, and the null mask is preserved below

Reply:
Wow! Thank you so much @tustvold , you are indeed making me better Rust programmer. Really appreciate for review. 👍